packages/clerk-js/src/ui/components/OrganizationProfile/ActiveMembersList.tsx

```diff
@@ -57,7 +56,12 @@ export const ActiveMembersList = () => {
       itemCount={organization.membersCount}
       itemsPerPage={ITEMS_PER_PAGE}
       isLoading={!membershipList}
-      headers={['User', 'Joined', 'Role', '']}
+      headers={[
+        localizationKeys('organizationProfile.membersPage.activeMembersTab.headerTitle__user'),
```

Comment:
Do you think renaming `headerTitle__user` to `tableHeader__user` makes its intention a little more apparent?
```
        localizationKeys('organizationProfile.membersPage.activeMembersTab.headerTitle__user'),
```

That way its a little harder to confuse with `headerSubtitle__members` that we're using below, which is the actual header subtitle of the page

Reply:
Same applies to the other relevant keys of course